backend/migrations/017-526-start-date-required-for-meal-plan.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE app.meal_plan ADD COLUMN startDate TEXT
```

Comment:
Please change it to start_date as that is the convention for postgres. Date type should be date

Reply:
done !